### PR TITLE
[KIECLOUD-149] Fix rhdm/rhpam trial environments

### PR DESCRIPTION
**Pull request by ruromero**
Fix https://issues.jboss.org/browse/KIECLOUD-149

* Disable JGROUPS for DecisionManager and BusinessCentral in trial environments

Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

### CODE REVIEW

File: config/envs/rhdm-trial.yaml [hunk 1]
```diff
@@ -7,6 +7,11 @@ console:
           spec:
             containers:
               - name: "[[.ApplicationName]]-[[.Console.Name]]"
+                env:
+                  - name: KIE_SERVER_HOST
```
Comment (sutaakar) [inline config/envs/rhdm-trial.yaml]: What is the purpose of specifying KIE_SERVER_HOST into console deployment config?

File: config/envs/rhpam-trial.yaml [hunk 2]
```diff
@@ -7,6 +7,11 @@ console:
           spec:
             containers:
               - name: "[[.ApplicationName]]-[[.Console.Name]]"
+                env:
+                  - name: KIE_SERVER_HOST
```
Comment (sutaakar) [inline config/envs/rhpam-trial.yaml]: Same as above.

File: config/envs/rhdm-trial.yaml (hunk 1 above)
Comment (ruromero) [reply config/envs/rhdm-trial.yaml]: Setting it without value will override the value from the common.yaml file

Comment (sutaakar) [reply config/envs/rhdm-trial.yaml]: Ok, thanks

Comment (sutaakar) [reply config/envs/rhdm-trial.yaml]: But why is it specified for Workbench? AFAIK Workbench has nothing to do with KIE_SERVER_HOST.

Comment (ruromero) [reply config/envs/rhdm-trial.yaml]: Because it's wrong and shouldn't be there :wink:

**Pull request merged**